### ISSUE TITLE
Invok-GPOZaurr after install

### ISSUE THREAD

**Issue by faris-cloud** [EvotecIT/GPOZaurr#26]
After installation and verifying dependencies are installed, unable to run module, as it states "Invoke-GPOAzurr" is not recognized as a name of a cmdlet, function, script file, or executable program.'

Running in Windows Server 2016. Same results when running in PS v5 v6 and v7

![2021-10-05 12_52_02](https://user-images.githubusercontent.com/23544796/136067958-2b7f5443-5bae-4bce-8c5e-f7989517ebd1.png)
.

**Comment by PrzemyslawKlys** [comment]
try Windows PowerShell. I believe GPO module doesn't support PowerShell 7 properly so it's limited to PS 5.1

**Comment by faris-cloud** [comment]
Error when running in PS v5.1

![2021-10-05 12_59_40](https://user-images.githubusercontent.com/23544796/136068859-9c7ef09e-c7f2-4611-8233-92eecfdb35e5.png)

**Comment by PrzemyslawKlys** [comment]
Get-Module -ListAvailable GPOZaurr 

Keep in mind that when you install module in PowerShell 7 it won't work automagically show up in PS 5.1. Those are stored in two different locations so you have to install module in 2 places if you've module that works in both. In case of GPOZaurr only install it on PS 5.1, and remove in PS 7.

**Comment by faris-cloud** [comment]
I think something else in the environment is limiting installing the module, which is part of the reason why I am trying to run the tool.  Will keep troubleshooting with you tho.

![2021-10-05 13_04_29](https://user-images.githubusercontent.com/23544796/136069584-ad7df84b-857c-4592-b25c-cc071c975382.png)
ti

**Comment by PrzemyslawKlys** [comment]
Import-Module GPOZaurr -Force -Verbose

**Comment by PrzemyslawKlys** [comment]
Get-Module -ListAvailable GroupPolicy

Please make sure you have RSAT for AD/GPO

**Comment by faris-cloud** [comment]
![2021-10-05 13_07_09](https://user-images.githubusercontent.com/23544796/136070022-cf0c12b3-1324-422a-97fe-709caa429468.png)

**Comment by PrzemyslawKlys** [comment]
Ye, I think you are missing GroupPolicy module.

**Comment by faris-cloud** [comment]
Yep. That's it. I ran the commands, but my account does not have the appropriate permissions to run/install optional features.

**Comment by PrzemyslawKlys** [comment]
Import-Module GroupPolicy -Verbose -Force
Import-Module GPOZaurr -Verbose -Force

Does it work now?

**Comment by faris-cloud** [comment]
Still not working. the GPOZaurr function appears to be missing?

**Comment by PrzemyslawKlys** [comment]
When the function is missing that means that either GroupPolicy or ActiveDirectory module failed to load. There is a logic that checks that ActiveDirectory and GroupPolicy modules it only exposes some commands (I've defined which ones show up). 

![image](https://user-images.githubusercontent.com/15063294/136076250-2e8d5114-baf1-4389-a926-e59ff0c646e5.png)

so that means that PSM1 doesn't see either GroupPolicy module or ActiveDirectory.

**Comment by faris-cloud** [comment]
I finally figured out what the issue was.  ActiveDirectory-PowerShell was needed.  Apparently, having it installed required a reboot.  It shouldn't have, but after reboot Invoke-GPOZaurr worked.

**Comment by sapeurpac** [comment]
I'm currently having issues with GPOZaurr : the GP module and the AD module don't seem to want to load. 
![image](https://github.com/user-attachments/assets/8da775c5-ac9c-421f-8012-b562275e8b40)

I've tried downloading the various modules with : 
`Add-WindowsCapability -Online -Name 'Rsat.ActiveDirectory.DS-LDS.Tools~~~~0.0.1.0'
Add-WindowsCapability -Online -Name 'Rsat.GroupPolicy.Management.Tools~~~~0.0.1.0'`
which doesn't seem to cause any problems, normal behavior but GPOZaurr continued trouble ...
(i tried many version of RSAT) So I tried again through the GUI, which brought nothing new.

![image](https://github.com/user-attachments/assets/023e54cf-d745-4554-bd9b-da35718c259a)
![image](https://github.com/user-attachments/assets/df339c21-ea11-43ff-ae52-bb1f2566f25f)

is there a solution to make GPOZaurr work without rebooting my server? (prod server) if no option is possible I would schedule a time slot dedicated to my tests hoping that the reboot will solve my problems ...

any help is welcome, thank you in advance for your time
